packages/roosterjs-editor-types/lib/index.ts

```diff
@@ -33,6 +33,7 @@ export { default as EditorReadyEvent } from './event/EditorReadyEvent';
 export { default as EntityOperationEvent } from './event/EntityOperationEvent';
 export { default as ExtractContentWithDomEvent } from './event/ExtractContentWithDomEvent';
 export { default as PendingFormatStateChangedEvent } from './event/PendingFormatStateChangedEvent';
+export { default as StartCropEvent } from './event/StartCropEvent';

```

Comment:
We should try to make an event as common as possible, but not just for one scenario.

Here this is used for plugin to know that we want to start crop. But normally we don't use event to just talk to one special plugin, because all other plugins will also receive it but they never need to handle this event. So we need to rethink about how should we notify a plugin that user is about to do something.